.github/workflows/development_infrastructure.yml

```diff
@@ -493,6 +510,12 @@ jobs:
             az cosmosdb mongodb collection show   --account tfs-${{ env.environment }}-mongo --database-name ${{ env.database_name }} --name feedback || \
             az cosmosdb mongodb collection create --account tfs-${{ env.environment }}-mongo --database-name ${{ env.database_name }} --name feedback --shard "_id"
 
+            az cosmosdb mongodb collection show   --account tfs-${{ env.environment }}-mongo --database-name ${{ env.database_name }} --name utilisationData || \
```

Comment:
Nothing to do with the review just sharing an opinion:
I'd much rather we defined stuff like this (db schema) in our application code on startup of central API or something rather than here in the infra